src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
```

Comment:
The names `threeDayDiscount` and `sevenDayDiscount` could be more descriptive. According to the checklist, use constants that explain the values they represent. For example, `SHORT_TERM_DISCOUNT` and `LONG_TERM_DISCOUNT` would be more descriptive and consistent with the good example provided in the checklist item 2.